types/aphrodite.d.ts

```diff
@@ -1,6 +1,29 @@
 declare module "aphrodite" {
     import * as React from "react";
 
+    const xs = "@media screen and (max-width: 567px) /* breakpoints.xs */";
```

Comment:
I really don't like having to hard code these values in here. But I tried _many_ different approaches to typing the tokens and this was the only thing that worked!

Reply:
Agreed, that's one of the things where TS could improve for sure!